cmd/digest/digest.go

```diff
@@ -0,0 +1,45 @@
+package digest
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	defaultDigestFile string = "/kaniko/.docker/digest-file"
+)
+
+func GetDigestFileName(digestFile, outputFile string) (string, error) {
```

Comment:
This is not required since we won't take digest file path from user

Reply:
removed